meson.options

```diff
@@ -0,0 +1,26 @@
+# Build the HTML / PDF documentation
```

Comment:
Please add the comments under the `description` field, then they are shown also by the meson cli.